### PR TITLE
Gihan

### PR DESCRIPTION
**Pull request by GihanAyesh**
Add or delete job titles to db function. Need to fix db side logic. Not sure about the changeJob function in employeeRecord.service. i think it is a mistake. Check just in case. 
Does salary required for deleteJob? 
What about job has role function ?

### CODE REVIEW

File: src/services/admin.service.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    return new JobTitle(
+        job
+    );
+  }
+  async removeJobTitle(job) {
+
+    const jobRepo = new JobTitleRepository(this.db);
+
+    await jobrepo.delete({
+      job
```
Comment (thamidurm) [inline src/services/admin.service.js]: meka jobTitle : job wenna one. Delete method eken wenne me query eka run weana eka.'Delete WHERE job_title = job';
mulin thiyena eka snake case ekata convert wenawa.

File: src/routes/controllers/admin.controller.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+const AdminService = require('../../services/admin.service');
+const db = require('../../db');
+
+class AdminController{
+    static async addJobTitle(req,res){
+        const jobTitle = req.body;
```
Comment (thamidurm) [inline src/routes/controllers/admin.controller.js]: should be req.body.jobTitle or something

File: src/services/admin.service.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  constructor(db) {
+    this.db = db;
+  }
+
+  async addJobTitle(job) {
+
+    const jobRepo = new JobTitleRepository(this.db);
+
+    await jobrepo.create({
+      job
```
Comment (thamidurm) [inline src/services/admin.service.js]: jobTitle : job

File: src/services/admin.service.js (hunk 1 above)
Comment (thamidurm) [reply src/services/admin.service.js]: line31

**Comment by thamidurm** [comment]
I think changeJob should be removed.

Salary isn't needed for deletion.

Job has role table is needed only for Yasiru's part

File: src/services/admin.service.js [hunk 4]
```diff
@@ -11,16 +11,18 @@ class AdminService {
...
   }
 
-  async addJobTitle(job) {
+  async addJobTitle(job,salary) {
 
     const jobRepo = new JobTitleRepository(this.db);
 
     await jobrepo.create({
-      job
+      job,
```
Comment (thamidurm) [inline src/services/admin.service.js]: Line 19 should change to title : job

**Comment by GihanAyesh** [comment]
All right now ?

**Pull request merged**